app/main.py

```diff
@@ -1 +1,39 @@
-# write your code here
+from typing import Any
+
+
+class Animal:
+    alive = []
+
+    def __init__(self, name: str, health: int = 100,
+                 hidden: bool = False, *args: Any) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = hidden
+        Animal.alive.append(self)
+
+    def die(self) -> None:
+        if self in Animal.alive:
+            Animal.alive.remove(self)
+
+    def __repr__(self) -> str:
+        return (f"{{Name: {self.name}, "
+                f"Health: {self.health}, "
+                f"Hidden: {self.hidden}}}")
+
+    def __str__(self) -> str:
+        return str([repr(animal) for animal in Animal.alive])
```

Comment:
The `__str__` method is intended to provide a string representation of the object. However, in this implementation, it returns a string representation of all alive animals, which might not be the expected behavior when calling `str()` on a single `Animal` instance. Consider revising this method to return a more appropriate string representation for individual instances.